src/diffpy/utils/parsers/serialization.py

```diff
@@ -40,13 +40,13 @@ def serialize_data(
 
     Parameters
     ----------
-    filename
-        Name of the file whose data is being serialized.
+    filepath
+        The file path whose data is being serialized.
     hdata: dict
-        File metadata (generally related to data table).
+        The file metadata (generally related to data table).
```

Comment:
is there a reason we are not calling this variable `metadata`?  or `header_data`.   calling it `hdata` I am having to carry around too many pointers in my brain that I have to cognitively resolve as I read.

Reply:
The hdata came from loadData's naming of hdata for header data information. Good to change to metadata.